src/pages/Landing/index.tsx

```diff
@@ -118,7 +118,7 @@ export default () => {
             onChange={(value) => setNewUser({ ...newUser, isCool: value })}
             isChecked={newUser.isCool}
           />
-          <Button type='submit'>Submit</Button>
+          <Button type='submit'>Snazzy</Button>
```

Comment:
This is not using your new component. You just changed the "Submit" buttons text to "Snazzy"